analyzers/tests/SonarAnalyzer.UnitTest/TestCases/RedundantDeclaration.CSharp9.cs

```diff
@@ -29,8 +38,10 @@ public class RedundantDeclaration
 {
     public RedundantDeclaration()
     {
-        MyEvent += new((a, b) => { }); // Noncompliant {{Remove the explicit delegate creation; it is redundant.}}
-//                 ^^^^^^^^^^^^^^^^^^
+        MyEvent += new((a, b) => { });
```

Comment:
could you also add a test with `(unused1, used, unused2) => { Foo(used);}` to cover this mixed case?